components/server/src/projects/projects-service.ts

```diff
@@ -239,9 +239,11 @@ export class ProjectsService {
                 await db.storeProject(project);
 
                 await this.auth.addProjectToOrg(installer.id, teamId, project.id);
+                await this.auth.setProjectVisibility(installer.id, project.id, teamId, "org-public");
             });
         } catch (err) {
             await this.auth.removeProjectFromOrg(installer.id, teamId, project.id);
+            await this.auth.setProjectVisibility(installer.id, project.id, teamId, "private");
```

Comment:
The PR description says project visibility is "org-public" is the default, but here it's "private"? I think this is bc we want to remove all relations, but might warrant a comment (at least for future me :upside_down:)

Reply:
Yes. I can also make the removal part of removeProjectFromOrg which seems better